src/Stack.CPP

```diff
@@ -0,0 +1,37 @@
+/*------------------------------------------------------------------------*/
+/*                                                                        */
+/*  STACK.CPP                                                             */
+/*                                                                        */
+/*  Copyright Borland International 1991                                  */
```

Comment:
s this really your code? I do not want to accept code which has been taken without consent.